ATAOS/v1/hex_m1_201912.yaml

```diff
@@ -0,0 +1,27 @@
+correction_frequency: 0.2
+
+m1:
```

Comment:
Please add a comment saying what these numbers are (and units if there are any).

Reply:
Do we want comments on the configuration? You can find that information on the schema file. https://github.com/lsst-ts/ts_ataos/blob/develop/schema/ATAOS.yaml

Reply:
Ok you're right. It shouldn't be in two places.